test/src/test/java/org/corfudb/benchmark/CorfuSingleNodeBenchmarkState.java

```diff
@@ -0,0 +1,181 @@
+package org.corfudb.benchmark;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.nio.NioEventLoopGroup;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.Getter;
+
+import org.corfudb.infrastructure.AbstractServer;
+import org.corfudb.infrastructure.BaseServer;
+import org.corfudb.infrastructure.CorfuServerNode;
+import org.corfudb.infrastructure.LayoutServer;
+import org.corfudb.infrastructure.LogUnitServer;
+import org.corfudb.infrastructure.SequencerServer;
+import org.corfudb.infrastructure.ServerContext;
+import org.corfudb.infrastructure.ServerContextBuilder;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.CorfuRuntime.CorfuRuntimeParameters;
+import org.corfudb.util.NodeLocator;
+
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+
+/**
+ * State for a Corfu benchmark. Initializes a simple single-server instance.
+ * Benchmark states for {@link AbstractCorfuBenchmark} should extend from this class.
+ *
+ * Created by zlokhandwala on 5/3/19.
+ */
+@State(Scope.Benchmark)
+public class CorfuSingleNodeBenchmarkState {
+
+    /**
+     * A single boss group for the trial.
+     */
+    @Getter
+    EventLoopGroup bossGroup;
+
+    /**
+     * A single worker group for the trial.
+     */
+    @Getter
+    EventLoopGroup workerGroup;
+
+    /**
+     * A single client group for the trial.
+     */
+    @Getter
+    EventLoopGroup clientGroup;
+
+    /**
+     * A single runtime group for the trial.
+     */
+    @Getter
+    EventLoopGroup runtimeGroup;
+
+    /**
+     * A map of servers, updated on each iteration.
+     */
+    @Getter
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651477)